var/spack/repos/builtin/packages/meson/package.py

```diff
@@ -30,4 +31,5 @@ class Meson(PythonPackage):
     # By default, Meson strips the rpath on installation. This patch disables
```

Comment:
Can you add a description to the variant?

Reply:
To be honest, I am not quite sure what it is for. While `MesonPackage` adds dependencies on both `meson` and `ninja`, hybrid packages like `atk` only have a meson dependency and trying to build them with `meson~ninjabuild` actually fails. Should I just remove it and make `ninja` a hard dependency?

Reply:
Yeah, if ninja is the only backend we can support, I would remove that variant.

Reply:
I have pushed a new commit that removes the variant.